### PR TITLE
add wemakedevs and hashnode swags

### LINKED ISSUE #1072: [Hacktoberfest] WeMakeDevs and Hashnode swags and accessories opportunity every month
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : WeMakeDevs and Hashnode

Q. Where have you seen the announcement (URL)?  
A. : In twitter (X): https://wemakedevs.org/events/hashnode

Q. What kind of swag (clothing, stickers, ...)?  
A. : Clothing and accessories

Q. Have you received swag from this?  
A. : Yes

Q. How long did it take?  
A. : One month

Q. Does it ship worldwide?  
A. : Yes

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by aashishpanthi**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I've added an opportunity to get accessories and swags from hashnode by writing blogs on hashnode every week with #wemakedevs. This fixes #1072

### CODE REVIEW

**Comment by aslafy-z** [comment]
Can you please rebase your pull request? Thank you

**Comment by aashishpanthi** [comment]
Also, can you add hacktoberfest label?

**Comment by aslafy-z** [comment]
The hacktoberfest label is added at the repository level, there is no need for it to be added to the pull request AFAIK.

You can find online guides on how to rebase, you can also merge the new version of master into your PR, or open a new one on the updated code directly.

**Comment by aashishpanthi** [comment]
@aslafy-z I think you can merge it now.

**Comment by aslafy-z** [comment]
Please check the "All the CI / unit" GitHub Actions result and update the link to make it happy! Thank you

**Comment by aashishpanthi** [comment]
@aslafy-z the reference link is valid. You can check that before merging. IDK why CI/unit failed

File: data.json [hunk 1]
```diff
@@ -782,6 +782,15 @@
     "dateAdded": "2020-09-30T16:35:00.000Z",
     "tags": ["hacktoberfest", "clothing", "stickers", "expired"]
   },
+  {
+    "name": "WeMakeDevs",
+    "difficulty": "medium",
+    "description": "Get a chance to win exciting prizes such as MacBooks, iPhones, iPads and more every month by writing blogs on hashnode with the tag #wemakedevs.",
+    "reference": "https://wemakedevs.org/events/hashnode",
```
Comment (aslafy-z) [inline data.json:789]: See `curl https://wemakedevs.org/events/hashnode` returns a 404.  Let's try with `https://wemakedevs.org/?/events/hashnode`

Comment (aslafy-z) [reply data.json:789]:
```suggestion
    "reference": "https://wemakedevs.org/?/events/hashnode",
```

**Comment by aslafy-z** [comment]
It looks like it's due to Cloudflare rejecting requests from GitHub Actions. I will see what can be done.

**Comment by aslafy-z** [comment]
So @aashishpanthi, I've *fixed* the issue. However I just seen that this opportunity is in fact a raffle. This kind of offers are out of the scope of DevSwag.io. Feel free to contribute another opportunity. Regards.

**Pull request closed without merging**